packages/sveltekit/package.json

```diff
@@ -50,6 +50,7 @@
     "sorcery": "0.11.0"
   },
   "devDependencies": {
+    "@babel/types": "7.20.7",
```

Comment:
why do we need the dependency here? 
(I don't think it's a problem per sé, just curios)

Reply:
`sveltekit/src/vite/autoInstrument.ts` is using the babel types, and as Angular upgraded those babel version, there was an type error.